test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/BusinessTradeNameViewSpec.scala

```diff
@@ -70,15 +72,18 @@ class BusinessTradeNameViewSpec extends ViewSpec {
     "have a hint text for textInput" in new Setup {
       document.mustHavePara(BusinessTradeNameMessages.hintText)
     }
-    "have a continue button" in new Setup {
+    "have a continue button when not in edit mode" in new Setup {
       document.getSubmitButton.text mustBe BusinessTradeNameMessages.continue
     }
+    "have update button when in edit mode" in new Setup(true) {
+      document.getSubmitButton.text mustBe BusinessTradeNameMessages.update
+    }
     "have a backlink " in new Setup {
       document.getBackLink.text mustBe BusinessTradeNameMessages.backLink
       document.getBackLink.attr("href") mustBe testBackUrl
     }
```

Comment:
Could you add another test similar to this, which will test the back link goes to the correct place when in edit mode?

Reply:
Nevermind, not how the code works, ignore this